### PR TITLE
Add ss-twitter

### PR DESCRIPTION
**Pull request by techguydave**
Displays a tweet with entrance and exit animations. Includes any images in the tweet.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -60,6 +60,7 @@ Want to suggest a bundle for inclusion? Open a PR!
 * [lfg-hypetrain](https://github.com/SupportClass/lfg-hypetrain) - Provides an API for tracking "hype trains", chains of events occuring one after another such as a series of subscription events on a Twitch channel.
 * [lfg-filter](https://github.com/SupportClass/lfg-filter) - Filter words and email addresses based on a user-editable blacklist.
 * [lfg-nucleus](https://github.com/SupportClass/lfg-nucleus) - Meta bundle that ties together functionality from other "lfg" helper bundles.
+* [ss-twitter](https://github.com/splitscreen/ss-twitter) - Displays a tweet as a graphic, including images. Based on TotH 2015.
```
Comment (Lange) [inline README.md]: Not everyone will know what "TotH 2015" is. What about something like "based on [toth3-overlay](https://github.com/TipoftheHats/toth3-overlay), the live graphics package developed for [Tip of the Hats 2015](http://tipofthehats.org/)"?

Comment (techguydave) [inline README.md]: That sounds good. I can update it when I get home tonight.

**Comment by techguydave** [comment]
Updated with the discussed change.

**Comment by Lange** [comment]
Looks good, thanks!

**Pull request merged**